R/derive_summary_records.R

```diff
@@ -125,18 +165,33 @@
 #' ) %>%
 #'   arrange(USUBJID, AVISIT)
 derive_summary_records <- function(dataset,
+                                   dataset_add = NULL,
```

Comment:
`dataset_add` should be mandatory to be consistent with the other functions.

Reply:
Yes, there are some implications but if implement `dataset_add` as optional, we should do it in all other functions as well. And this could have much more implications.
It's a pity that we do not have more time.